IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormFileHistoryTests.cs

```diff
@@ -39,6 +39,47 @@ public void File_history_should_behave_as_expected(string fileRelativePath, stri
             }, Path.Combine(fileRelativePath, fileName), commands);
     }
 
+    [Test]
+    public void File_history_should_display_contents_in_all_revisions()
+    {
+        const string text = "This text will appear in a file that shall be renamed";
+
+        using ReferenceRepository referenceRepository = new(createCommit: false);
```

Comment:
Currently there's only one test in this class, so the current implementation suffices. However, in a test class with more than one test using `ReferenceRepository` we should be doing this:
```cs
        // Created once for the fixture
        private ReferenceRepository _referenceRepository;

        // Created once for each test
        private GitUICommands _commands;

        [SetUp]
        public void SetUp()
        {
            ReferenceRepository.ResetRepo(ref _referenceRepository);
            _commands = new GitUICommands(GlobalServiceContainer.CreateDefaultMockServiceContainer(), _referenceRepository.Module);
        }

        [OneTimeTearDown]
        public void OneTimeTearDown()
        {
            _referenceRepository.Dispose();
        }

```